hack/release.sh

```diff
@@ -377,7 +371,7 @@ main() {
 	setup_s3
 	setup_gpg
 	release_binaries
-	release_ubuntu
+	release_repos
```

Comment:
I think repos should just be taken out since @jfrazelle runs the new repos separately, but she'll have to confirm that.